src/applications/vaos/components/PendingAppointment.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router';
+import moment from 'moment';
+
+const timeText = {
+  AM: 'in the morning',
+  PM: 'in the afternoon',
+  'No Time Selected': '',
+};
+
+function formatDate(date) {
+  const parsedDate = moment(date, 'MM/DD/YYYY');
+
+  if (!parsedDate.isValid()) {
+    return '';
+  }
+
+  return parsedDate.format('MMMM D, YYYY');
+}
+
+export default function PendingAppointment({ appointment }) {
+  const isCommunityCare = appointment.typeOfCareId.startsWith('CC');
```

Comment:
Not sure if this is the best way to tell if something is CC, need to go verify that in the current app.